src/esp/bindings_js/webvr.html

```diff
@@ -21,7 +21,13 @@ <h1>Habitat</h1>
     <div id="status">Initialization...</div>
     <div id="status-description"></div>
   </div>
+
+
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/webxr-polyfill@latest/build/webxr-polyfill.js"></script>
```

Comment:
We should double check this doesn't cause a CORS error on our website. Thoughts @eundersander . Worst case we could always bundle it.

Reply:
Seems fine to me.

I put a question to our Slack dev channel about how our website gets updated, e.g. https://aihabitat.org/demo/ .

But I don't think @BlaiseRitchie 's changes here will get automatically pushed to our website. If I'm correct, then your concern about CORS error is more about if/when anyone tries to push this app to a website (right?), so I guess we could defer the question and testing until then.

Linking to this jsdelivr URL seems to be good practice according to https://github.com/immersive-web/webxr-polyfill .